modin/core/execution/modin_aqp.py

```diff
@@ -84,8 +86,19 @@ def call_progress_bar(result_parts, line_no):
     bar_lock.release()
 
     threading.Thread(target=_show_time_updates, args=(progress_bars[pbar_id],)).start()
+
+    modin_engine = Engine.get()
+    if modin_engine == "Ray":
+        from ray import wait
+    elif modin_engine == "Unidist":
```

Comment:
We probably need to enable a test for this engine?

Reply:
We don't have any test even for Ray engine, do we? As far as I remember, someone who added the support for progress bar just tested it manually because we need to see how the progress bar is performing.

Reply:
Ok for now. https://github.com/modin-project/modin/issues/5495